java/src/test/java/org/hyperledger/fabric/client/InMemoryCheckpointerTest.java

```diff
@@ -0,0 +1,17 @@
+package org.hyperledger.fabric.client;
+
+
+import java.io.IOException;
+
+public class InMemoryCheckpointerTest extends CommonCheckpointerTest {
+
+    @Override
+    protected InMemoryCheckpointer getCheckpointerInstance() throws IOException {
+        return new InMemoryCheckpointer();
+    }
+
+    @Override
+    protected void tearDown() throws IOException {
```

Comment:
Not a big deal; just to be tidy
```suggestion
    protected void tearDown() {
```